en/docs/concepts/compliance/gdpr.md

```diff
@@ -0,0 +1,226 @@
+
+---
+
+## How is a specialized IAM solution better at complying with regulations?
+
+Generally, identity data are scattered over several systems within an organization. In order to reach GDPR compliance, it is required to review, redesign, and modify each of these systems. This is a maintenance overhead and consumes a significant portion of annual IT budget and requires a specialized set of skills for continuous review
+and modification process. Considering the amount of data available wiithin an organization and the intensity of adverse impact on the organization in case of failure to comply with GDPR, skilled staff working on a centralized, secure environment for data processing is vital. 
+
+A GDPR-compliant IAM solution ensures that all identity profiles are managed centrally and shares only required data with other systems in an on-demand manner through well-known security standards such as [SAML](TO DO: insert-concepts) and [OpenIDConnect](TO DO: insert-concepts). Any efficient IAM solution supports anonymization to remove PII data from datasets, and pseudonymization to set aritifical identifiers to uniquely identify the user, instead of their personal data. This mitigates the risk of exposing the personal data of individuals to compromised environments. 
+
+
+---
+
+## How does WSO2 Identity Server ensure compliance?
+
+WSO2 Identity Server (WSO2 IS) is designed based on privacy best practices, and is fully compliant with GDPR. GDPR compliance in IAM and API security spaces can be completely fulfilled with WSO2 IS.
+
+WSO2 IS has been designed and architectured based on well known *Secure by Design* and *Privacy by Design* principles.  With the
+formalization of GDPR in 2016, the product architecture of WSO2 Identity Server has been reviewed and fine-tuned accordingly to support privacy principles in an efficient manner, with less overhead to product performance and user experience. WSO2 IS provides all privacy features enabled in the product as default options, and uses up-to-date
+algorithms and frameworks for all cryptographic operations such as data encryption, signing, and hashing.
+
+WSO2 IS is subjected to regular reviews and updates for latest versions
+of the crypto algorithm and latest versions of crypto frameworks. These
+security updates are provided as WSO2 WUM services. Additionally, a
+number of data encryption and protection features are supported by WSO2
+IS.
+
+Following are the encryption features supported for personal data.
+
+-   OAuth2 Access token
+
+-   OAuth2 Refresh token
+
+-   OAuth2 Authorization
+
+-   ID Tokens
+
+-   SAML Responses
```

Comment:
maybe we can link these to their corresponding concept docs? I think we have pages on ID Tokens, refresh token etc.

Reply:
The concept docs do not speak about the encryption part so it might be confusing. Information is more relatable in the articles so I have mentioned them in the related topics.